renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
Since these scripts relate closely to the scripts that are affected by the parallel build PR ([mup-web#321](https://github.com/meetup/mup-web/pull/321)), I've made the name changes there. I think these are really good updates for clarifying what goes where, although it has made the mup-web PR a bit larger. I think that was very close to being mergeable, but hopefully that can happen soon so there are fewer big file moves and var name changes hanging around.